lib/app/data/providers/get_storage_provider.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:get/get.dart';
+import 'package:get_storage/get_storage.dart';
+
+class GetStorageProvider extends GetxService{
+  String? lanCode;
```

Comment:
I understand why you've done this, but I think it would be cleaner to make functions similar to how we've implemented in secure_storage_provider since we'll probably move some functions from there to get_storage_provider as well. You'd have to update the way you handle this in the main.dart file as well.

Reply:
Right, on it

Reply:
I've removed the language controller, moved language functionalities to settings, and made the language.dart file cleaner by separating other language translations into separate files.
I've checked the persistence of language preference by killing the app and then opening it again, the chosen language preference persists.
GetStorageProvider is more generalised now...
Do let me know if anything else is required!